features/focus-events.yml

```diff
@@ -0,0 +1,17 @@
+name: Focus events
+description: The `blur`, `focus`, `focusin` and `focusout` events fire when there are changes in focus for the elements or windows involved.
+spec: https://w3c.github.io/uievents/#events-focusevent
+
+# Caniuse vs BCD is conflicting, so not adding the Caniuse field nor
+# computing status here yet until figured out in this issue:
+#
+# https://github.com/web-platform-dx/web-features/issues/2327
+
```

Comment:
I think we can negotiate our way into a useful-now status, but it's tricky. We have to write the description to dodge the question a bit and compute from `blur` and `focus`.

(Also, I tried really hard to replace "gets or loses focus" with something actually descriptive, but I can't find an actual definition of "focus." A real mystery.)

```suggestion
name: Focus events
description: Focus events, such as `focus` and `blur`, fire when an element gets or loses focus.
spec: https://w3c.github.io/uievents/#events-focusevent
compute_from:
  - api.Element.blur_event
  - api.Element.focus_event
# It's possible that caniuse's `focusin-focusout-events` could be linked here
# or broken out into a separate feature. But caniuse and BCD conflict,
# making this difficult. See this issue to help resolve this:
# https://github.com/web-platform-dx/web-features/issues/2327
```